primefaces/src/main/resources/META-INF/resources/primefaces/forms/forms.commandbutton.js

```diff
@@ -23,6 +25,18 @@ PrimeFaces.widget.CommandButton = PrimeFaces.widget.BaseWidget.extend({
         PrimeFaces.skinButton(this.jq);
 
         this.bindTriggers();
+
+        if (cfg.validateClientDynamic) {
+            let that = this;
+
+            // init enabled/disabled-state after initial page-load
+            setTimeout( function() {
```

Comment:
should we use the new task api @melloware introduced?

Reply:
Yes please use `PrimeFaces.queueTask`

Reply:
Thanks for pointing me on this. Reminded there was some replacement for window.setTimeout but could not figure out myself what anymore.